ibis/expr/operations/relations.py

```diff
@@ -373,6 +373,15 @@ def _projection(self):
         return Projection(self.table, self.selections)
 
 
+@public
+class DummyTable(TableNode):
+    ops = rlz.nodes_of(rlz.scalar(rlz.any), min_length=1)
```

Comment:
How about the following:

```py
values = rlz.nodes_of(rlz.any, min_length=1)
```

I assume we could use this for unbound column expressions as well.

Reply:
Nah, this is only for modeling `SELECT` statements that don't require a `FROM`, like `SELECT 1 + 1` or `SELECT 'a'`, etc.